ansible/ml-data-pipeline.yml

```diff
@@ -0,0 +1,6 @@
+---
+- hosts: ml-data-pipeline
+  become: yes
+  roles:
+    - role: ml-data-pipeline
+      BASEPATH: "/opt/sparkjobs" # Git Clone Path 
```

Comment:
Sorry what is this? I don't recall anything ansible inbuilt function in this name. If its a variable, please add under the `vars:` section.

Reply:
i have removed it because it is mentioned in defaults